packages/m2m-gateway/src/model/errors.ts

```diff
@@ -107,3 +107,13 @@ export function purposeVersionNotFound(
     title: "Purpose version not found",
   });
 }
+
+export function missingActivePurposeVersion(
+  purposeId: string
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `There is no active version for purpose ${purposeId}`,
+    code: "missingActivePurposeVersion",
+    title: "Missing active purpose version",
+  });
+}
```

Comment:
We use this when looking for a draft version so the name/message are misleading

```suggestion
export function missingActivePurposeVersionWithState(
  purposeId: string,
  state: purposeApi.PurposeState
): ApiError<ErrorCodes> {
  return new ApiError({
    detail: `There is no ${satate} version for purpose ${purposeId}`,
    code: "missingActivePurposeVersionWithState",
    title: `Missing ${satate} purpose version`,
  });
}
```